src/System.Runtime.Extensions/src/System.Runtime.Extensions.csproj

```diff
@@ -17,6 +17,8 @@
     <NuGetTargetMoniker Condition="'$(NuGetTargetMoniker)' == ''">.NETStandard,Version=v1.7</NuGetTargetMoniker>
     <!-- copy to uap10.1 until nuget makes the change to map that to ns1.7 -->
     <PackageTargetFramework Condition="'$(TargetGroup)' == ''">netstandard1.7;uap10.1</PackageTargetFramework>
+    <!-- Temporarily ignore missing types for uap101aot -->
+    <GenFacadesArgs Condition="'$(TargetGroup)' == 'uap101aot'">$(GenFacadesArgs) -ignoreMissingTypes</GenFacadesArgs>
```

Comment:
I am getting the following errors if this flag is omitted:
Did not find type 'T:System.AppDomain' in any of the seed assemblies.
Did not find type 'T:System.AssemblyLoadEventArgs' in any of the seed assemblies.
Did not find type 'T:System.AssemblyLoadEventHandler' in any of the seed assemblies.
Did not find type 'T:System.Base64FormattingOptions' in any of the seed assemblies.
Did not find type 'T:System.LoaderOptimization' in any of the seed assemblies.
Did not find type 'T:System.LoaderOptimizationAttribute' in any of the seed assemblies.
Did not find type 'T:System.MathF' in any of the seed assemblies.
Did not find type 'T:System.Progress`1' in any of the seed assemblies.
Did not find type 'T:System.ResolveEventHandler' in any of the seed assemblies.